### PR TITLE
user params now wired in to google-trends-api query

### PR DESCRIPTION
**Pull request by brendanrsmith**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -141,15 +143,21 @@ async function domain(array) {
...
+  const startTime = new Date(reqBody.startTime);
+  const endTime = new Date(reqBody.endTime);
+  const geo = reqBody.geo;
+  console.log(keyword, startTime, endTime, geo);
+  return await googleTrends.relatedQueries({keyword: keyword, startTime: startTime, endTime: endTime, geo: geo})
     .then(results => {
       const parsedResults = JSON.parse(results);
       //possible creat a toggle button that switches from 0 to 1 based on what the user is looking for.
       const relatedKeyword = parsedResults.default.rankedList[0].rankedKeyword;
+      console.log(relatedKeyword);
```
Comment (JasonDormier) [inline server.js:158]: tsk tsk

**Review by JasonDormier — APPROVED** [review]
swellerific

File: server.js (hunk 1 above)
Comment (brendanrsmith) [reply server.js:158]: still in for debugging.

**Review by Elliron — APPROVED** [review]
Looks good

**Pull request merged**